x/stake/keeper.go

```diff
@@ -614,6 +614,13 @@ func (k Keeper) setNewParams(ctx sdk.Context, params Params) {
 	store.Set(ParamKey, b)
 }
 
+// Public version of setNewParams
+func (k Keeper) SetNewParams(ctx sdk.Context, params Params) {
```

Comment:
vital: all write functions should be part of the `PrivledgedKeeper` in staking not the regular Keeper

Reply:
This isn't implemented yet, right?

Reply:
correct - this comment is n/a